hack/ci/start-kind.sh

```diff
@@ -18,24 +18,24 @@
 
 set -e
 
-if [ -z "${KUBE_VERSION}" ]; then
-    KIND_KUBE_VERSION=""
-else
-    KIND_KUBE_VERSION="--image kindest/node:${KUBE_VERSION}"
-fi
+#Make sure docker is installed before proceeding
+which docker > /dev/null || (echo "docker is not installed. Please install it before proceeding exiting...." && exit 1)
```

Comment:
Not trying to be obnoxious, but marking it so none are left behind :-)

You should still be able to keep a similar syntax:

```suggestion
command -v docker &> /dev/null || (echo "docker is not installed. Please install it before proceeding exiting...." && exit 1)
```